<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>337</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SolaimanLipi</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>পণ্য জমা/খরচ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lddate">
     <property name="placeholderText">
      <string>তারিখ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="font">
      <font>
       <family>SolaimanLipi</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rb_cash">
       <property name="text">
        <string>নগদ</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rb_rice">
       <property name="text">
        <string>চাউল</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_oil">
       <property name="text">
        <string>তেল</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_oilcake">
       <property name="text">
        <string>খৈল</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_sesame">
       <property name="text">
        <string>তিল</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_mustard">
       <property name="text">
        <string>সরিষা</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="ld_quantity">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>পরিমান (কেজি)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ld_rate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>দর (৳)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="ldamount">
       <property name="placeholderText">
        <string>টাকার পরিমান</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_calculate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>গণনা</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="tddesc">
     <property name="placeholderText">
      <string>মন্তব্য</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_msg">
     <property name="text">
      <string>শেষ সংযোজনঃ নেই</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QRadioButton" name="rb_add_more">
      <property name="text">
       <string>আরো যোগ করুন</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
